Replace material imports in the devtools with the legacy definitions

lukasmatta opened the pull request:
In the PR [#48216](https://github.com/angular/angular/pull/48216) Material deps were updated to v15, but the components and modules in the DevTools were not replaced with the legacy ones nor updated to the new MDC-based ones.

Resolves #48419

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

HyperLife1119 commented:
As an official tool, I hope it can migrate to the new MDC component instead of continuing to adopt the legacy component.

lukasmatta committed:
fix(devtools): Replace material imports and styles

In the #48216 Material deps were updated to v15,
but the components and modules in DevTools were not
updated to MDC nor replaced with the legacy definitions.

AleksanderBodurri reviewed (approved):
Looks good to me @lukasmatta. Thank you for catching this 🙏 

@HyperLife1119 I agree. For now though let's rely on the legacy components and migrate over to the MDC ones in a separate issue/PR.

jessicajaniuk commented:
This PR was merged into the repository by commit 2e65a2bd8433bf5656968d78a55b96ad878f9561.
